app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -52,10 +52,11 @@ const VpcNameFromId = ({ value }: { value: string }) => {
   // possible because you can't delete a VPC that has child resources, but let's
   // be safe
   if (isError) return <Badge color="neutral">Deleted</Badge>
-  if (!vpc) return <Spinner /> // loading
+  if (!vpc)
+    return <div className="h-4 w-12 rounded bg-tertiary motion-safe:animate-pulse" /> // loading
```

Comment:
Using a skeleton state here rather than the full spinner which can be a bit strong.

Reply:
I didn't know we had that. That's great. You should update `SubnetNameFromId` too, which is also using the spinner. Also maybe make it wider (maybe not the pulsing thing itself, but its container) so the pop-in is less severe when the name shows up? Though that depends on making a guess about the average length of name.